mk/PcapPlusPlus.mk.common

```diff
@@ -16,5 +16,14 @@ PCAPPP_POST_BUILD :=
 PCAPPP_BUILD_FLAGS :=
 
 ifdef PCAPPP_ENABLE_CPP_FEATURE_DETECTION
-	PCAPPP_BUILD_FLAGS += -DPCAPPP_CPP_FEATURE_DETECTION -std=c++11 
+	PCAPPP_BUILD_FLAGS += -DPCAPPP_CPP_FEATURE_DETECTION -std=c++11
 endif
+
+ifndef CFLAGS
+CFLAGS := -O2 -g -Wall -c
+endif
+
+ifndef CXXFLAGS
+CXXFLAGS := -O2 -g -Wall -c
+endif

```

Comment:
I just recalled that we already have `PCAPPP_BUILD_FLAGS`. Do we really need `CFLAGS` and `CXXFLAGS` as well?

Reply:
`CFLAGS` and `CXXFLAGS` are `make` variables and are commonly used. Also, OSSFuzz [requires them](https://google.github.io/oss-fuzz/getting-started/new-project-guide/#Requirements) to be used.

If you prefer using only `PCAPPP_BUILD_FLAGS` internally in the `Makefiles` and such, I can assign the value of `CXXFLAGS` to `PCAPPP_BUILD_FLAGS` after their assignment here.

Reply:
Yes, I think that would be a better option